### ISSUE TITLE
Incorrect UTC support

### ISSUE THREAD

**Issue by pcboy** [ebeigarts/exchanger#11]
When I'm using the CalendarItem, EWS is returning CalendarEvents like this:

```
          <CalendarEvent>
            <StartTime>2013-08-26T06:43:53</StartTime>
            <EndTime>2013-08-26T06:53:53</EndTime>
            <BusyType>Busy</BusyType>
          </CalendarEvent>
```

The exchanger library is [calling Time.xmlschema](https://github.com/ebeigarts/exchanger/blob/ec0b5bdbd1eb1a4332d0d4999734e2a402318e35/lib/exchanger/field.rb#L133) on these times.
The problem is that they are not in the correct xmlschema format. These times are UTC. But the [xmlschema specification](http://www.w3schools.com/schema/schema_dtypes_date.asp) is clear, for UTC times you need to add a Z at the end of it.

See:

```
2.0.0-p247 :003 > Time.xmlschema '2013-08-26T06:43:53'
=> 2013-08-26 06:43:53 +0900 
2.0.0-p247 :004 > Time.xmlschema '2013-08-26T06:43:53Z'
 => 2013-08-26 06:43:53 UTC 
```

The first one is converted to my local timezone. The second one is correct.
So basically, if you book meetings in UTC, you'll have a bad time when you get them back.
Am I wrong?
